src/quill-cursors/cursor.ts

```diff
@@ -106,6 +132,28 @@ export default class Cursor {
     selections.forEach((selection: ClientRect) => this._addSelection(selection, container));
   }
 
+  private _setHoverState(): void {
+    this._caretEl.classList.add(Cursor.CONTAINER_NO_POINTER_CLASS);
+    document.addEventListener('mouseover', this._toggleOpenedCursor);
```

Comment:
We're still getting the flag "stuck" as per my [previous comment](https://github.com/reedsy/quill-cursors/pull/80#pullrequestreview-931583407). However, I think we can fix it if we change this to listen for `mousemove` instead:

```suggestion
    document.addEventListener('mousemove', this._toggleOpenedCursor);
```